Credify/Credify/WebView/WebViewController.swift

```diff
@@ -30,7 +30,11 @@ class WebViewController: UIViewController {
         let configuration = WKWebViewConfiguration()
         let userController = WKUserContentController()
             
-        presenter.handlers.forEach { handler in
+        presenter.sendHandlers.forEach { handler in
```

Comment:
Is this necessary? I think this is not needed.

Reply:
I removed it and it still works well.